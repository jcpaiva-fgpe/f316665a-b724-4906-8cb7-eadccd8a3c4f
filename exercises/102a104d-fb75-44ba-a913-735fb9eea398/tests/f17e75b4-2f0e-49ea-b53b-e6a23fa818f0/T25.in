67{[h+{(i-43(99o+97((r-84u+19g)-53r))+54{15l+74u}-32(([z+6(47(69c+41t)+{k-j})]-m)+85p))+87([47k+r]-i)+53n}+35p+16g]-(64{{31[9d+51i+31y]-50g}-5(b+58v)-62{m-y}}-72x-83s-z)-r}+89u
