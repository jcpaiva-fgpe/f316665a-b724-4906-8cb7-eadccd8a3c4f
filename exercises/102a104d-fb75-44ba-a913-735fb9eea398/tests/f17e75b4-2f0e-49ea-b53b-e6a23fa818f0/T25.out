() 20 30
() 19 35
() 12 36
{} 40 48
() 61 69
{} 71 75
() 58 76
[] 54 77
() 53 80
() 52 85
() 7 86
[] 91 97
() 90 100
{} 6 105
[] 3 114
[] 123 134
{} 120 139
() 142 148
{} 152 156
{} 119 157
() 116 168
{} 2 171
